aws/resource_aws_instance_test.go

```diff
@@ -4886,16 +4899,7 @@ resource "aws_subnet" "test" {
 }
 
 func testAccAwsInstanceVpcConfigBasic(rName string) string {
-	return fmt.Sprintf(`
-data "aws_availability_zones" "available" {
-  state            = "available"
-
-  filter {
-    name   = "opt-in-status"
-    values = ["opt-in-not-required"]
-  }
-}
-
+	return testAccAvailableAZsNoOptInConfig() + fmt.Sprintf(`
```

Comment:
Nit: could also use `composeConfig` to concatenate these 2 configs

Reply:
Switched to use this!